build_linux.sh

```diff
@@ -21,23 +21,23 @@ export CODEC2DIR=$FREEDVGUIDIR/codec2
 export LPCNETDIR=$FREEDVGUIDIR/LPCNet
 
 # change this when working on combined codec2/freedv-gui changes
-CODEC2_BRANCH=master
-LPCNET_BRANCH=master
+CODEC2_BRANCH=v1.0.3
+LPCNET_BRANCH=v0.2
```

Comment:
So if we are checking out codec2 v1.0.3 ... how do we get 2020B?

Reply:
The user would need to edit this script to use master instead (or link against a Codec2 built manually from it).